### PR TITLE
Zoom to Entity

### PR DESCRIPTION
**Pull request by SwedishSock**
Added a function to move the entity editor's camera to the currently selected entity and looking at it, to easier make your way to whichever entity you'd like.

### CODE REVIEW

**Comment by NinjaDanz3r** [comment]
Can one of the admins verify this patch?

File: src/Editor/GUI/Editors/EntityEditor.hpp [hunk 1]
```diff
@@ -40,6 +40,12 @@ namespace GUI {
              */
             void SetEntity(Entity* entity);
 
+            /// Get the entity
```
Comment (Chainsawkitten) [inline src/Editor/GUI/Editors/EntityEditor.hpp]: "Get the entity being edited" would be more descriptive.

File: src/Editor/GUI/ResourceList.hpp [hunk 2]
```diff
@@ -36,6 +36,9 @@ namespace GUI {
             
             /// Reset which scene is open.
             void ResetScene();
+
+            /// Return Scene Editor
```
Comment (Chainsawkitten) [inline src/Editor/GUI/ResourceList.hpp]: Document return value.

File: src/Editor/GUI/ResourceList.hpp [hunk 3]
```diff
@@ -36,6 +36,9 @@ namespace GUI {
...
+            SceneEditor GetScene();
```
Comment (Chainsawkitten) [inline src/Editor/GUI/ResourceList.hpp]: Return a reference rather than a copy of the object.

File: src/Editor/Editor.cpp [hunk 4]
```diff
@@ -106,26 +106,34 @@ void Editor::Show(float deltaTime) {
...
+
                 if (ImGui::MenuItem("Filters"))
                     filtersWindow.SetVisible(true);
-                
+
                 ImGui::EndMenu();
             }
-            
+
+            if (ImGui::Button("Move to current Entity")) {
```
Comment (Chainsawkitten) [inline src/Editor/Editor.cpp]: Maybe put this under "View"?

**Pull request merged**